index.js

```diff
@@ -20,14 +20,20 @@ function getRenderer(str, options, locals) {
   // given { define: { foo: 'bar', baz: 'quux' } }
   // runs renderer.define('foo', 'bar').define('baz', 'quux')
 
-  var allowed = ['set', 'include', 'import', 'define', 'use'];
+  var allowed = ['set', 'define'];
   var special = {}
+  var allowedSingle = ['include', 'import', 'use'];
+  var specialSingle = {}
   var normal = clone(options);
   for (var v in options) {
     if (allowed.indexOf(v) > -1) {
       special[v] = options[v];
       delete normal[v];
     }
+    else if (allowedSingle.indexOf(v) > -1) {
+        specialSingle[v] = options[v];
```

Comment:
Some extra whitespace here.

Otherwise, looks pretty good! Do the tests cover all these options?